ibex_configs.hjson

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Can we please use normal JSON or YAML here? hjson is a very OpenTitan thing [and actually not super active development now that JSON5 has arrived] and parsers are not as widely available as for JSON and for YAML. (I'd actually go for YAML, since that's the default config file format also for RISC-V DV and fusesoc.)

Reply:
I have switched to yaml